lib/src/modules/task.cc

```diff
@@ -108,6 +123,10 @@ Task::Task(const fs::path& exec_prefix,
 
     input_validator_.registerSchema(input_schema);
     results_validator_.registerSchema(output_schema);
+
+    client_.set_ca_cert(ca);
```

Comment:
I get an error trying to use this: `Failed to execute the task for the non blocking 'task run' request (transaction 142). Error: Downloading the task file init failed. Reason: SSL: Can't load the certificate "~/puppetlabs/orchestrator/dev-resources/ssl/cert.pem" and its private key: OSStatus -25299`.

Any ideas?